src/bindings.cpp

```diff
@@ -108,8 +108,15 @@ extern "C"
     /// and comes from the audio thread (so on the web, from a different web worker).
     FFI_PLUGIN_EXPORT void voiceEndedCallback(unsigned int *handle)
     {
+        bool isHandleFound;
         if (player != nullptr)
-            player->removeHandle(*handle);
+        {
+            isHandleFound = player->findByHandle(*handle) != nullptr;
+            if (isHandleFound)
+                player->removeHandle(*handle);
+            else
+                return;
```

Comment:
This is getting a bit too complex for me and I'd love to have a short comment here. Something like:

```suggestion
                // When player exists but handle isn't found,
                // bail out immediately.
                return;
```

So, when player _isn't_ null and we call this callback, we continue executing this function?

Reply:
As a more general point, maybe elaborate on the logic here, because it's a bit too dense for my brain. The fact that there's a return statement in one of the branches makes it hard to reason about, imho. But maybe it's just a matter of elaborating on the comment suggested above?